### PR TITLE
ENH- i18n template values and user-comprehensible strings for upload status

### PR DESCRIPTION
**Pull request by rksh**
Making the uploads screen display something people can read for status instead of single-letter codes.

### CODE REVIEW

File: wiglewifiwardriving/src/main/res/values-cs/strings.xml [hunk 1]
```diff
@@ -256,5 +256,11 @@
     <string name="disable_toast">Disable toast pop-ups</string>
     <string name="activate">aktivujte nyní</string>
     <string name="setting_activate_prompt">visit https://wigle.net/activate for your user code</string>
-
+    <string name="upload_queued">Queued</string>
+    <string name="upload_parsing">Parsing...</string>
+    <string name="upload_trilaterating">Trilaterating...</string>
+    <string name="upload_stats">Final Stats...</string>
```
Comment (bobzilladev) [inline wiglewifiwardriving/src/main/res/values-cs/strings.xml]: could use the elipse single character, might make it fit a little better. oh hell, this is on a million files tho. i18n ftl.

**Comment by bobzilladev** [comment]
krad, nice polish, +27b/6

File: wiglewifiwardriving/src/main/res/values-cs/strings.xml (hunk 1 above)
Comment (rksh) [inline wiglewifiwardriving/src/main/res/values-cs/strings.xml]: done as suggested

**Comment by bobzilladev** [comment]
I'm Lovin' It

**Pull request merged**